pysiaf/utils/compare.py

```diff
@@ -8,6 +8,7 @@
     dict_compare was adapted from
     https://stackoverflow.com/questions/4527942/comparing-two-dictionaries-in-python/4527957
 """
+
```

Comment:
I think PEP8 wants no blank line after """ (but one before). It does not matter for this PR and I know I have been inconsistent in what concerns code style and documentation.